.travis.yml

```diff
@@ -9,3 +9,5 @@ node_js:
   - '11'
   - '10'
   - '8'
+before_script:
+  - cd examples && npm install && cd ../guide && npm install && cd ..
```

Comment:
Some of the examples use external dependencies which aren't in the top-level `package.json`, so we have to install them before running `tsc` in Travis. (Otherwise TypeScript throws errors for missing dependencies.)

Reply:
thanks!